app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -374,4 +374,39 @@ class LuciBuildService {
     final GetBuildRequest request = GetBuildRequest(id: id, fields: fields);
     return buildBucketClient.getBuild(request);
   }
+
+  /// Reschedules a prod build using [commitSha], [builderName], [branch] and
+  /// [repo]. Default values for [branch] is "master" and default for [repo] is
+  /// "flutter". This method return a boolean with true if the build was
+  /// rescheduled or false if it was not scheduled.
+  Future<bool> rescheduleProdBuild({
```

Comment:
See a couple of existing functions like scheduleBuild, rescheduleBuild. Are they all general for both `try` and `prod`? If there are cases specifically for `try`, then it would be good to add `try` key word in the function names to reflect.

Reply:
Done